lib/main_ltp.pm

```diff
@@ -104,7 +105,21 @@ sub stress_snapshots {
     }
 }
 
-sub maybe_load_kernel_tests {
+sub is_kernel_test {
+    return ( get_var('INSTALL_LTP')
+          || get_var('LTP_SETUP_NETWORKING')
+          || get_var('LTP_COMMAND_FILE')
+          || get_var('INSTALL_KOTD')
+          || get_var('QA_TEST_KLP_REPO')
+          || get_var('INSTALL_KOTD')
+          || get_var('VIRTIO_CONSOLE_TEST'));
+}
+
+sub load_kernel_tests {
+    if (is_kernel_test()) {
```

Comment:
This has been checked already. Either keep it as maybe_load_kernel_tests or remove the inner check.